common/reviews/api/tools.api.md

```diff
@@ -41,7 +41,7 @@ type ActorSliceRange = {
 };
 
 // @public (undocumented)
-function addAnnotation(element: HTMLDivElement, annotation: Annotation): string;
+function addAnnotation(annotation: Annotation, element?: HTMLDivElement): string;

```

Comment:
I agree this is probably a good change, but that is definitely a breaking change.  Is there a second name we could add/implement to make it backwards compatible for a while or is it the time still for breaking changes?
Is there a good reason to include the element even optionally?  If it is just for frameOfReferenceUID, then that could be applied directly, or maybe should already be inside annotation?  My vote is to remove it entirely, assuming it has to be in annotation directly.

Reply:
Yes, this is a breaking change and I want it to be before 1.x. 
Element is optional since we can have element specific annotation manager (which we haven't implemented yet)